src/components/provider/show/provider-show.test.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import {
+  render,
+  cleanup,
+  fireEvent,
+} from '@testing-library/react';
+
+import {
+  CommandList,
+  defaultKeyboardShortcuts,
+} from '@folio/stripes-components';
+
+import ProviderShow from './provider-show';
+import Harness from '../../../../test/jest/helpers/harness';
+import { collapseAllShortcut } from '../../../../test/jest/utilities';
+import { buildStripes } from '../../../../test/jest/__mock__/stripesCore.mock';
+
+jest.mock('@folio/stripes/smart-components', () => ({
+  ...jest.requireActual('@folio/stripes/components'),
+  NotesSmartAccordion: ({ open }) => (open ? (<span>content of NotesSmartAccordion</span>) : null),
```

Comment:
Maybe it will be better to add `NotesSmartAccordion` into existing `stripesSmartComponent.mock.js`?

Reply:
Nice catch, thank you. Fixed.